TestControllers/AzureController.psm1

```diff
@@ -190,4 +190,64 @@ Class AzureController : TestController
 			}
 		}
 	}
+
+    [void] GetSystemBasicLogs($AllVMData, $User, $Password, $CurrentTestData, $CurrentTestResult, $enableTelemetry) {
```

Comment:
Code duplicates for different platforms. My suggestion is to keep the Get-SystemBasicLogs in TestLogs.psm1, but move GetSQLQueryOfTelemetryData to TestProvider.psm1, then in TestController.psm1, call TestProvider.GetSQLQueryOfTelemetryData right after where Get-SystemBasicLogs is called

Reply:
or add a new method ReportTelemetryData to TestProvider.psm1, which get the sql query and upload to database.
This method can be overwritten by Providers of different platforms.
And it's called after Get-SystemBasicLogs, in TestController

Reply:
done